Tests/Support/Snapshots/discriminator/Sources/Entities/ContainerWithDirectOneOf.swift

```diff
@@ -0,0 +1,42 @@
+// Generated by Create API
+// https://github.com/CreateAPI/CreateAPI
+
+import Foundation
+
+public enum ContainerWithDirectOneOf: Codable {
+    case a(A)
+    case three(Three)
+    case four(Four)
```

Comment:
This `case four(Four)` becomes `case string(String)` without the change in this PR